### PR TITLE
add component Media card

### PR DESCRIPTION
**Pull request by chienit3bk**

### CODE REVIEW

File: src/components/MediaCard/MediaCard.vue [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    div(
+      :class="mediaContainerClassName",
+    )
+      slot
+    div(
+      :class="infoContainerClassName",
+    )
+      CardSection
+        div(
+          v-if="(popoverActions.length > 0)",
```
Comment (blacklen) [inline src/components/MediaCard/MediaCard.vue:15]: no parentheses

File: src/components/MediaCard/utils.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import type { IconableAction, DisableableAction, BadgeAction, DestructableAction } from '@/utilities/interface';
+export interface ActionListItemDescriptor extends IconableAction, DisableableAction, BadgeAction, DestructableAction {
```
Comment (blacklen) [inline src/components/MediaCard/utils.ts:2]: already exist on ActionList component

File: src/components/MediaCard/MediaCard.vue [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                size="slim",
+                :plain="true",
+                :accessibilityLabel="lang.Polaris.MediaCard.popoverActivatorLabel",
+              )       
+        Stack(
+          :vertical="true",
+          spacing="tight",
+        )
+          div(
+            :class="styles.Heading"
```
Comment (HQCuong) [inline src/components/MediaCard/MediaCard.vue:42]: missing commna or you can code as follow `div(:class="styles.Heading)`

File: src/components/MediaCard/MediaCard.vue [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+          )
+            Heading {{ title }}
+          p(
+            :class="styles.Description"
```
Comment (HQCuong) [inline src/components/MediaCard/MediaCard.vue:46]: same here

File: src/components/MediaCard/MediaCard.vue [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  styles.InfoContainer,
+  props.portrait && styles.portrait,
+  props.size === 'small' && styles.sizeSmall,
+);
+
+</script>
+
+<style lang="scss">
+@import "polaris/polaris-react/src/components/MediaCard/MediaCard.scss";
+</style>
```
Comment (HQCuong) [inline src/components/MediaCard/MediaCard.vue:139]: lint new line at end of file

File: src/components/MediaCard/utils.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    prefixId?: string;
+    /** Suffix source */
+    suffixId?: string;
+    /**  Add an ellipsis suffix to action content */
+    ellipsis?: boolean;
+    /** Whether the action is active or not */
+    active?: boolean;
+    /** Defines a role for the action */
+    role?: string;
+}
```
Comment (HQCuong) [inline src/components/MediaCard/utils.ts:19]: lint new line at end of file

File: src/components/MediaCard/MediaCard.vue [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+  popoverActive.value = !popoverActive.value;
+};
+const props = withDefaults(defineProps<Props>(), {
+  popoverActions: () => [],
+  size: 'medium',
+  portrait: false,
+});
+const lang = inject('lang') as Record<string, any>;
+
+const actionClassName = classNames(
```
Comment (HQCuong) [inline src/components/MediaCard/MediaCard.vue:113]: Should this code be computed? Same for the code below

**Pull request merged**